src/errors/helper/assertionErrorHelper.js

```diff
@@ -0,0 +1,19 @@
+const REQUIRED_PARAMENTER_MISSING = 'REQUIRED_PARAMENTER_MISSING';
```

Comment:
```suggestion
const REQUIRED_PARAMENTER_MISSING_ERROR = 'REQUIRED_PARAMENTER_MISSING_ERROR';
```
Doe we have a naming convention for the names?

Reply:
this is only the detail, not the error(-type)